modes/basic-dev-mode/src/toolbarButtons.js

```diff
@@ -54,147 +23,91 @@ function _createWwwcPreset(preset, title, subtitle) {
   };
 }
 
-const toolbarButtons = [
-  // Measurement
+function _createSetToolActiveCommands(toolName, toolGroupIds = ['default', 'mpr', ]) {
+  return toolGroupIds.map(toolGroupId => ({
+    commandName: 'setToolActive',
+    commandOptions: {
+      toolGroupId,
+      toolName,
+    },
+    context: 'CORNERSTONE',
+  }));
+}
+
+const toolbarButtons: Button[] = [
   {
     id: 'MeasurementTools',
-    type: 'ohif.splitButton',
+    uiType: 'ohif.splitButton',
     props: {
-      groupId: 'MeasurementTools',
-      isRadio: true, // ?
-      // Switch?
-      primary: _createToolButton(
-        'Length',
-        'tool-length',
-        'Length',
-        [
-          {
-            commandName: 'setToolActive',
-            commandOptions: {
-              toolName: 'Length',
-            },
-            context: 'CORNERSTONE',
-          },
-        ],
-        'Length'
-      ),
+      groupId: 'MeasurementToolsGroupId',
+      evaluate: 'evaluate.group.promoteToPrimaryIfCornerstoneToolNotActiveInTheList',
+      primary: ToolbarService.createButton({
```

Comment:
Could you import the createButton function - eg:
const { createButton } = ToolbarService

The reason is it allows you to over-ride it for anything specific that you want to do.